tests/5.0/loop/test_loop_order_concurrent.F90

```diff
@@ -0,0 +1,81 @@
+!===--- test_loop_order_concurrent.F90 -------------------------------------===//
+!
+! OpenMP API Version 5.0 Nov 2018
+!
+! This test checks the loop directive with the order(concurrent) clause.
+! The order(concurrent) clause is assumed to be present if it is not
+! present, so this test covers the standalone loop directive as well. The
+! test creates a parallel region with a loop construct nested within, and
+! performs simple operations on an int array which are then checked for
+! correctness. Additionally, since loop binds to a parallel region, the test
+! checks randomly that other threads wait before proceeding out of the loop
+! region. The number of threads is checked in the parallel region but after
+! the loop construct because runtime API calls are not permitted in loop
+! directive regions.
+!
+!//===----------------------------------------------------------------------===//
+
+#include "ompvv.F90"
+
+#define N 1024
+
+PROGRAM test_loop_order_concurrent
+  USE iso_fortran_env
+  USE ompvv_lib
+  USE omp_lib
+  implicit none
+
+  OMPVV_TEST_OFFLOADING
+
+  OMPVV_TEST_VERBOSE(test_loop() .ne. 0)
```

Comment:
change to OMPVV_TEST_AND_SET_VERBOSE()

Reply:
In this case I'm just using TEST_VERBOSE since there's no need for an explicit errors variable, this will generate and error if the condition isn't met